server/src/test/java/org/elasticsearch/index/store/FsDirectoryFactoryTests.java

```diff
@@ -42,7 +42,7 @@ public void testPreload() throws IOException {
         doTestPreload("*");
         Settings build = Settings.builder()
             .put(IndexModule.INDEX_STORE_TYPE_SETTING.getKey(), IndexModule.Type.HYBRIDFS.name().toLowerCase(Locale.ROOT))
-            .putList(IndexModule.INDEX_STORE_PRE_LOAD_SETTING.getKey(), "dvd", "bar")
+            .putList(IndexModule.INDEX_STORE_PRE_LOAD_SETTING.getKey(), "dvd", "tmp")
```

Comment:
why way this change necessary? `bar` is not a valid extension, just as `tmp`?

Reply:
`bar` is not a valid extension while `tmp` is used by Lucene for temporary files. I've added

> // Temporary Lucene file
> TMP("tmp", "Temporary File", false, false),

for this purpose